new/chef_env_attribute.md

```diff
@@ -0,0 +1,41 @@
+---
+RFC: unassigned
+Title: Expose more settings as attributes
+Author: Thom May <thom@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Exposing some core settings as attributes
+
+There are a number of settings, such as the node name and the environment, that we've historically not exposed as automatic attributes.
+The environment attribute is a bit more complicated in the world of
+policy groups, and this RFC clarifies that situation too
```

Comment:
Can we give an example of how you'd reference this as an attribute vs. the "old" way.

Reply:
Not sure that the audience should need that level of explanation?